src/app/tests/suites/tests.js

```diff
@@ -864,15 +864,15 @@ function getTests() {
     ];
 
     const DoorLock = [
-        "DL_UsersAndCredentials",
+        //"DL_UsersAndCredentials", TODO: This test is not aligned with spec
         "DL_LockUnlock",
         "DL_Schedules",
         "Test_TC_DRLK_2_2",
         "Test_TC_DRLK_2_3",
         "Test_TC_DRLK_2_4",
         "Test_TC_DRLK_2_5",
         "Test_TC_DRLK_2_7",
-        "Test_TC_DRLK_2_9",
+        //"Test_TC_DRLK_2_9", TODO: This test is not aligned with spec
```

Comment:
Please make sure there is a follow-up to align this YAML with the test plan, at least, and re-enable it here.